experiment_scheduler/db_util/connection.py

```diff
@@ -0,0 +1,26 @@
+"""
+This file exists to manage sqlalchemy function
+"""
+import ast
+
+from sqlalchemy import create_engine, MetaData
+from sqlalchemy.orm import declarative_base, sessionmaker
+from sqlalchemy_utils import database_exists, create_database
+from experiment_scheduler.common.settings import USER_CONFIG
+
+db_url = ast.literal_eval(USER_CONFIG.get("default", "db_url"))
+Base = declarative_base()

```

Comment:
Setting Session as 'session' much intuitive.
Base is okay since it used as class, but session is not.
If you have values both you do and don't want to export, try use underscore

```python
_db_url = "db_url"
Base = declarative_base() 
_engine = create_engin(_db_url, echo=False)
session = sessionmaker(bind=engine)
```